docs/network-edge/edges.mdx

```diff
@@ -0,0 +1,115 @@
+# Edges
+
+---
+
+## Overview
+
+Edges enable you to centrally manage your endpoints' Module configurations in
+the ngrok dashboard or API instead of defining them via an [Agent](/agent/) or
+[Agent SDK](/agent-sdks). With Edges, you can update those configurations
+without taking your endpoints offline.
+
+With Edges, you start your agent once and then configure it via the dashboard
+or API. When you make changes to your Edge's configuration by adding or
+updating modules, your app doesn’t go offline. You never have to restart the
+agent to pick up new command line flags or configuration files.
+
+Edges enable you to:
+
+- Enhance your endpoints with [Modules](#modules) using the dashboard or API
+  without restarting the Agent.
+- Centrally manage your module configurations in ngrok's cloud service. This
+  decouples the configuration of module behavior on your endpoint from the agents
+  where traffic is routed to.
+- Load balance traffic to different ngrok agents
+- Apply different configurations for each path in your application. For
+  example, you may apply OAuth on `/app` and Compression on `/static/css`.
+- Use the [Kubernetes Ingress Controller](/k8s/). The Ingress Controller is
+  powered by Edges.
+
+When you create an Edge, you must select the protocol corresponding to the
+endpoints you would like it to create: one of HTTPS, TLS or TCP.
+
+For additional detail on per-protocol Edge behavior, see:
+
+- [HTTPS Edges](/http/#edges)
+- [TLS Edges](/tls/#edges)
+- [TCP Edges](/tcp/#edges)
+
+## Modules
+
+Modules are middleware that enable you manipulate traffic to your applications.
+Modules enable to add authentication, resiliency, acceleration, transformation,
+security and observability to the traffic that transits through to your
+applications.
```

Comment:
```suggestion
Modules are middleware that enable you to manipulate traffic to your applications.
Modules can provide authentication, resiliency, acceleration, transformation,
security and observability to the traffic that transits through to your
applications.
```
mostly avoiding redundant "enable" here. "traffic .. to your applications" is also a bit repetitive too though. 

In general, this section feels a little lightweight and abstract without any examples of modules here to round out the reader's understanding. I'd maybe recommend directing the reader to the protocol-specific edge pages to see what modules they offer, but that seems a bit repetitive with the few lines above this section. 🤷🏻‍♀️